platform/broadcom/sai.dep

```diff
@@ -5,7 +5,7 @@ DEP_FILES   := $(SONIC_COMMON_FILES_LIST) platform/broadcom/sai.mk platform/broa
 DEP_FILES   += $(SONIC_COMMON_BASE_FILES_LIST)
 # Get the Latest HTTP Header and calculate the SHA value as it is a softlink that always points to LATEST_INT_OCP_SAI_X.X.X
 SAI_FLAGS   := $(shell wget --spider --server-response $($(BRCM_SAI)_URL) $($(BRCM_SAI_DEV)_URL)  2>&1 \
-			   | grep -Ev -- '--|Date:'|sha256sum|awk '{print $$1}' )
+			   | grep -Ev -- '--|Date:'| grep -v 'x-ms-request-id' |sha256sum|awk '{print $$1}' )
```

Comment:
>x-ms-request-id [](start = 41, length = 15)

You can combine it with previous regex, `|` means OR.

Reply:
As Qi mentioned, you can try as bellow.

| grep -Ev -- '--|Date:|x-ms-request-id'|sha256sum|awk '{print $$1}'